app/views/posts/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+	<div class="d-flex justify-content-center h-100">
+		<div class="card background post new">
+			<div class="card-header">
+				<h3>Create a new post</h3>
+			</div>
+
+			<div class="card-body">
+				<%= render partial: 'posts/form' %>
+			</div>
+			<div class="card-footer">
+				<%# <%= button_to "Add image", posts_path(@post), { method: :get, class: "buttonhome btn float-left font font-12 btnbackground blue" } %>

```

Comment:
Seems like this button is commented, if you don't need it then you can remove this div